### PR TITLE
Feat: Added helper for Indian English

### LINKED ISSUE #38: Add a helper method for Indian English
See #1 on how to implement it. It's very simple and it's the perfect candidate for the first issue collaboration.

Feel free to post any questions.

Code example: https://github.com/chgasparoto/aws-polly-voices/pull/2/files

### PR DESCRIPTION
**Pull request by yashguptaz**
Fix #38 
## Proposed changes

Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/chgasparoto/aws-polly-voices/blob/master/CONTRIBUTING.md) doc
- [ ] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by yashguptaz** [comment]
@chgasparoto Can you help me with the check?

File: src/index.js [hunk 1]
```diff
@@ -81,6 +81,13 @@ export default class Voices {
   polish () {
     return this.byLangCode('pl-PL')
   }
+  indianEnglish () {
+    return this.byLangCode('en-IN')
+  }
+
+  indian () {
+    return this.byLangCode('en-IN')
+  }
```
Comment (chgasparoto) [inline src/index.js]: Let's stick with only **indianEnglish** helper method for the Indian English language. Hindi is the only bilingual language in AWS Polly and needs special attention.

You can see the extra property in [voices.json](https://github.com/chgasparoto/aws-polly-voices/blob/master/src/voices.json#L179) file.

File: README.md [hunk 2]
```diff
@@ -154,6 +154,8 @@ const turkish = voices.turkish()
 const american = voice.american()
 const brazilian = voice.brazilian()
 const british = voices.british()
+const indian = voices.indian()
```
Comment (chgasparoto) [inline README.md]: Please change it to **indianEnglish**

File: test/index.test.js [hunk 3]
```diff
@@ -77,7 +77,15 @@ const mockVoices = [{
...
-}]
+},
+{
+  Gender: 'Male',
+  Name: 'Justin',
+  LanguageName: 'GB English',
+  Id: 'Justin',
+  LanguageCode: 'en-GB'
+}
+]
```
Comment (chgasparoto) [inline test/index.test.js]: Should be Indian data here, not GB English. That's why your unit tests are failing.

File: test/index.test.js [hunk 4]
```diff
@@ -330,4 +339,18 @@ describe('Voices', () => {
 
     spy.mockRestore()
   })
+
+  test('it should return INDIAN ENGLISH voice data', () => {
+    const spy = jest.spyOn(voices, 'byLangCode')
+    const indian = voices.indian()
```
Comment (chgasparoto) [inline test/index.test.js]: And here as well, change it to **indianEnlgish** please

File: src/index.js [hunk 5]
```diff
@@ -81,6 +81,13 @@ export default class Voices {
   polish () {
     return this.byLangCode('pl-PL')
   }
+  indianEnglish () {
```
Comment (chgasparoto) [inline src/index.js]: Insert a blank line here please, to sperate the methods.

**Comment by yashguptaz** [comment]
@chgasparoto 
I'm having some issues in handing the branch (with which I have created the PR) locally.

Can you help me with this?

image

I cannot switch to the Indian-English branch which I by mistake created using GitHub's website rather than my local machine.

**Comment by yashguptaz** [comment]
I have made all the changes and also figured out the above problem. I just needed to add the `--track` flag 👍

**Comment by yashguptaz** [comment]
@chgasparoto 
Let's merge this as soon as possible I want to close this PR out.

File: README.md [hunk 6]
```diff
@@ -154,6 +154,8 @@ const turkish = voices.turkish()
 const american = voice.american()
 const brazilian = voice.brazilian()
 const british = voices.british()
+const indianEnglish = voices.indian()
+
```
Comment (chgasparoto) [inline README.md]: could you remove this extra line?

**Comment by chgasparoto** [comment]
@yashguptaz I understand you would like to merge it asap but let's do the things in its time. We really don't need to rush it :)

**Pull request merged**